pkg/metrics/metrics.go

```diff
@@ -842,6 +848,15 @@ func NewLegacyMetrics() *LegacyMetrics {
 			},
 		}),
 
+		PolicyIncrementalUpdateDuration: metric.NewHistogramVec(metric.HistogramOpts{
+			ConfigName: Namespace + "_policy_incremental_update_duration",
+
+			Namespace: Namespace,
+			Name:      "policy_incremental_update_duration",
+			Help:      "Time between learning about a new identity and it being fully added to all policies.",
+			Buckets:   prometheus.ExponentialBuckets(10e-6, 10, 8),
+		}, []string{"scope"}),
```

Comment:
Maybe it helps to describe what the `scope` label means and its values?

Reply:
Adding this label was a mistake; I intend to get rid of the separate code paths anyways.